.travis.yml

```diff
@@ -11,8 +11,10 @@ addons:
 before_script: createdb lms_test; npm install;
 jobs:
   include:
-    - script: make test
-    - script: make lint
+    - env: ACTION=test
+      script: make test
+    - env: ACTION=lint
```

Comment:
The "ACTION=" lines have no effect on the build itself, but they make it easier to tell what the jobs are for when looking at the Travis status dashboard for a given build.